lib/util/http/gitlab.ts

```diff
@@ -83,4 +84,19 @@ export class GitlabHttp extends Http<GitlabHttpOptions> {
       throw err;
     }
   }
+
+  protected override calculateRetryDelay(retryObject: RetryObject): number {
+    const { error, attemptCount, retryOptions } = retryObject;
```

Comment:
```suggestion
  protected override calculateRetryDelay({ error, attemptCount, retryOptions }: RetryObject): number {
```
why not use destruction in function args, likle you do in base class ?

Reply:
Much better indeed, not sure why I haven't done it from the start 🤔

Reply:
Ah, now I remember: We need the complete `retryObject` for passing it to the base class.